packages/grid/_modules_/grid/constants/localeTextConstants.ts

```diff
@@ -82,4 +82,8 @@ export const GRID_DEFAULT_LOCALE_TEXT: GridLocaleText = {
 
   // Total rows footer text
   footerTotalRows: 'Total Rows:',
+
+  // Checkbox selection text
+  checkboxSelectionHeaderName: 'Checkbox Selection',
```

Comment:
This capitalization would be more consistent

```suggestion
  checkboxSelectionHeaderName: 'Checkbox selection',
```